### PR TITLE
Upgrade Compose to dev05.

### PR DESCRIPTION
**Pull request by zach-klippenstein**
Release notes:
 - https://developer.android.com/jetpack/androidx/releases/compose#0.1.0-dev05
 - https://developer.android.com/jetpack/androidx/releases/ui#0.1.0-dev05

### CODE REVIEW

File: kotlin/.buildscript/configure-compose.gradle [hunk 1]
```diff
@@ -19,6 +19,7 @@ android {
     compose true
   }
   composeOptions {
+    kotlinCompilerVersion "1.3.61-dev-withExperimentalGoogleExtensions-20200129"
```
Comment (tehras) [inline kotlin/.buildscript/configure-compose.gradle]: Just curious - why is this required?

Comment (zach-klippenstein) [reply kotlin/.buildscript/configure-compose.gradle]: The docs said to add it, but they were wrong about other things – I just removed it and it seems to work fine.
